include/CL/opencl.hpp

```diff
@@ -9392,9 +9392,9 @@ class CommandQueue : public detail::Wrapper<cl_command_queue>
                 object_, userFptr, args.first, args.second,
                 (mem_objects != nullptr) ? (cl_uint) mem_objects->size() : 0,
                 (mem_objects->size() > 0 ) ? reinterpret_cast<const cl_mem *>(mem_objects->data()) : nullptr,
-                (mem_locs != nullptr && mem_locs->size() > 0) ? (const void **) &mem_locs->front() : nullptr,
+                (mem_locs != nullptr && mem_locs->size() > 0) ? const_cast<const void**>(&mem_locs->front()) : nullptr,
```

Comment:
Needing to use `const_cast` here is unfortunate, but I don't think there is any way to avoid it.

Linking with https://github.com/KhronosGroup/OpenCL-Docs/issues/9 for completeness.